protected/humhub/modules/ui/form/widgets/DatePicker.php

```diff
@@ -105,4 +101,20 @@ public function run()
         JuiAsset::register($this->getView());
     }
 
+    private function registerLanguageAsset()
+    {
+
+        if($this->pickerLanguage === 'ru') {
+            DatePickerRussianLanguageAsset::register($this->getView());
+            return;
+        }
+
+        $assetBundle = DatePickerLanguageAsset::register($this->getView());
+        if(substr($this->pickerLanguage, 0 , 2) === 'en') {
+            $assetBundle->autoGenerate = false;
+            $assetBundle->js[] = "ui/i18n/datepicker-$this->pickerLanguage.js";
```

Comment:
`{$this->pickerLanguage}`?

Reply:
Not sure what you mean, this block did not change, did it? I just moved it to this function since the russian language condition was added.

Reply:
Ok, I changed it anyway, looked creepy.

Reply:
Ah ok,  on mobile it looked like you marked another line. Strange, but I just copied this block from the old version.